app/code/Magento/CatalogSearch/Controller/Advanced/Result.php

```diff
@@ -32,33 +25,35 @@ class Result extends \Magento\Framework\App\Action\Action
      *
      * @param Context $context
      * @param ModelAdvanced $catalogSearchAdvanced
-     * @param UrlFactory $urlFactory
```

Comment:
@ishakhsuvarov I will fix them. Seem that Travis build failed? Can you give your idea?